extensions/survey/src/survey.ts

```diff
@@ -0,0 +1,46 @@
+import { Util } from "@k8slens/extensions";
+import { machineId } from "node-machine-id";
```

Comment:
I noticed `node-machine-id` has this caveat:

Image-based environments have usually the same machine-id